evm_loader/lib/src/account_storage.rs

```diff
@@ -159,11 +159,10 @@ impl<'rpc, T: Rpc> Rpc for EmulatorAccountStorage<'rpc, T> {
         &self,
         pubkeys: &[Pubkey],
     ) -> ClientResult<Vec<Option<Account>>> {
-        // TODO: Optimize this!!!
-        let mut result = vec![];
-        for key in pubkeys {
-            let account = self.get_account(key).await?.value;
-            result.push(account);
+        let mut result: Vec<Option<Account>> = Vec::new();
+        for chunk in pubkeys.chunks(100) {
+            let mut accounts = self.rpc.get_multiple_accounts(chunk).await?;
```

Comment:
Some of pubkeys can be already in accounts_cache. What about them? We will request it again from the node?

Reply:
ah yea, updated to be same as `_get_account_from_rpc`, thanks